libmscore/midimapping.cpp

```diff
@@ -143,7 +143,7 @@ void MasterScore::rebuildExcerptsMidiMapping()
             for (Part* p : ex->partScore()->parts()) {
                   const Part* masterPart = p->masterPart();
                   if (!masterPart->score()->isMaster()) {
-                        qWarning("reorderMidiMapping: no part in master score is linked");
+                        qWarning() << QString("rebuildExcerptsMidiMapping: no part in master score is linked with %1").arg(p->partName());
```

Comment:
There is no need to use `.arg()` here, just successive `<<` operators would be enough:
```
qWarning() << "rebuildExcerptsMidiMapping: no part ..." << p->partName();
```

Reply:
I tried that, didn't work

Reply:
Looking again now...

Reply:
Apparently I didn't try that variant ;-)